tests/kernel/ib_tests.pm

```diff
@@ -39,18 +39,16 @@ sub check_dmesg {
 }
 
 sub ibtest_slave {
-    my $host1 = get_required_var('IBTEST_IP1');
-    my $host2 = get_required_var('IBTEST_IP2');
+    my $master = get_required_var('IBTEST_IP1');
+    my $slave  = get_required_var('IBTEST_IP2');
 
     # wait for master setup done
     barrier_wait('IBTEST_SETUP');
 
     assert_script_run('ssh-keygen -b 2048 -t rsa -q -N "" -f ~/.ssh/id_rsa');
-    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$host1");
-    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$host2");
-
-    # unload firewall. MPI- and libfabric-tests require too many open ports
-    script_run("systemctl stop firewalld");
+    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$master");
+    script_run("/usr/bin/clear");
+    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$slave");
```

Comment:
in this case, you are doing the ssh copy to the IP you think it's slave, but as I said above, it could be the master own IP.  Ok to do it to both master and slave to be in the safe way, but not needed if you know the real master ip :)